cicecore/cicedynB/dynamics/ice_dyn_vp.F90

```diff
@@ -328,7 +328,7 @@ subroutine implicit_solver (dt)
 
 ! tcraig, tcx, threading here leads to some non-reproducbile results and failures in icepack_ice_strength
 ! need to do more debugging
-      !$TCXOMP PARALLEL DO PRIVATE(iblk,ilo,ihi,jlo,jhi,this_block)
+      !$TCXOMP PARALLEL DO PRIVATE(iblk,ilo,ihi,jlo,jhi,this_block,ij,i,j)
```

Comment:
Is this loop still not working?  If it is, then remove TCX?

Reply:
The VP still has a few known problems.  One with this OMP loop, another with restarts, and possibly another with bit-for-bit with different decomps and/or OMP, https://github.com/CICE-Consortium/CICE/issues/518.